docs/tutorials/training_transformers.md

```diff
@@ -0,0 +1,93 @@
+# Training Transformers
+
+This tutorial shows how to train a transformer policy network.
+
+<div class="admonition note">
+<p class="admonition-title">Note: Understand how to launch training runs and view results first!</p>
+
+Before trying to train a transformer, it might be useful to read the following tutorials:
+- [how to launch training runs](./configs.html)
+- [how to view training results](./viewing_results.html)
+- [how to launch multiple training runs efficiently](./hyperparam_scan.html) 
+
+</div>
+
+A template with tuned parameters for transformer based policy networks is defined in `robomimic/config/default_templates/bc_transformer.json`.
+
+#### 1. Using default configurations
+
+The easiest way to train a transformer policy network is to pass the default template json to the main training script `train.py` via the `--config` argument. The dataset can be specified by setting the `data` attribute of the `train` section of the config json, or specified via the `--dataset` argument.
+
+```sh
+$ python train.py --config ../config/default_templates/bc_transformer.json --dataset /path/to/dataset.hdf5
+```
+
+#### 2. Defining custom parameters
+
+If you want to modify the default transformer parameters, do not directly modify the default config (`config/bc_config.py`) or template (`config/default_templates/bc_transformer.json`).  Instead, you should modify the default template in python code and then call the train function, as shown in the code snippet below.
+
+This code highlights useful parameters to tune for transformers and modifications that should be made when training a policy on observation images (default is low-dimensional states).  To see all transformer policy settings, refer to `config/bc_config.py`.
```

Comment:
"on observation images (default is low-dimensional states)" - change to "on image observations (default is low-dim observations)" to be consistent with other terminology in paper and in docs